Kwc/Abstract/Image/CropImage.js

```diff
@@ -227,9 +228,9 @@ Kwc.Abstract.Image.CropImage = Ext2.extend(Ext2.BoxComponent, {
                 cropData.width = this._userSelectedCropRegion.width;
                 var width = cropData.height * this.outWidth / this.outHeight;
                 var height = cropData.width * this.outHeight / this.outWidth;
-                if (width < this.width) {
+                if (width < Math.round(this.width * this.scaleFactor)) {
```

Comment:
maybe we should add a field imageWidth already containing value from this.width * this.scaleFactor
same for imageHeight